src/requests/features.jl

```diff
@@ -59,6 +55,16 @@ function get_definitions(x::EXPR, tls::StaticLint.Scope, server, locations)
     end
 end
 
+safe_isfile(s) = false
```

Comment:
What is the purpose of this method? I think it would be safer to see a method error if someone tries to call this function with something other than an `AbstractString`, because that always seems like a bug to me?

Reply:
@ZacLN could you take a look at this method here? I think once we have sorted this out, we could merge this?

Reply:
Removed this and added a `safe_isfile(s::Symbol)` method, because that one actually kinda makes sense.